permission_handler_platform_interface/lib/src/permissions.dart

```diff
@@ -95,42 +96,46 @@ class Permission {
   static const speech = Permission._(14);
 
   /// Android:
-  ///   On Android < TIRAMISU the `READ_EXTERNAL_STORAGE` and
-  ///   `WRITE_EXTERNAL_STORAGE` permissions are requested (depending on the
-  ///   definitions in the AndroidManifest.xml) file.
   ///   On Android TIRAMISU and higher this permission is deprecrated and
-  ///   always returns `PermissionStatus.denied`, instead use
-  ///   `Permission.photos`, `Permission.video`, `Permission.audio` or
-  ///   `Permission.manageExternalStorage`. For more information see our [FAQ](https://pub.dev/packages/permission_handler#faq).
-  /// iOS: Access to folders like `Documents` or `Downloads`. Implicitly
-  /// granted.
+  ///     always returns `PermissionStatus.denied`, instead use
+  ///     `Permission.photos`, `Permission.video`, `Permission.audio` or
+  ///     `Permission.manageExternalStorage`. For more information see our
+  ///     [FAQ](https://pub.dev/packages/permission_handler#faq).
+  ///   On Android < TIRAMISU the `READ_EXTERNAL_STORAGE` and
+  ///     `WRITE_EXTERNAL_STORAGE` permissions are requested (depending on the
+  ///     definitions in the AndroidManifest.xml) file.
```

Comment:
nit: what is the reason for the extra spaces here? It looks a bit strange as the first line is indented 2 spaces, while the second line (of the same sentence) is indented 4 spaces. 

If the idea is to make it clear there are two different options lets convert it into a list using an dash (`-`) or asterisk (`*`).